src/main/java/net/glowstone/net/pipeline/CodecsHandler.java

```diff
@@ -40,7 +40,7 @@ protected void encode(ChannelHandlerContext ctx, Message msg, List<Object> out)
 
         // write body
         ByteBuf messageBuf = ctx.alloc().buffer();
-        messageBuf = reg.getCodec().encode(messageBuf, msg);
+        messageBuf = reg.getCodec().encode(null, messageBuf, msg);
```

Comment:
We were thinking of reverting the changes in flow-network because of this. In retrospect, it doesn't make much sense for Codecs to have access to the session or context (that job should be left to the MessageHandlers). Passing `null` simply bypasses this new feature, which we aren't sure we need anyway.

Reply:
@momothereal I see. I assume that's why you left it on `1.2.5`, while things are discussed over on `flow-network`. If those changes are reverted, I don't think this PR is needed.

Reply:
Yeah, I updated the dependency without really checking if it had backwards-incompatible changes :) We've all been pretty busy which is why it's been left like this.

If you are still interested, feel free to apply a revert to https://github.com/GlowstoneMC/network/pull/1 on the `GlowstoneMC/network` repo. We'll then publish a release and update the Glowstone repo accordingly.